lib/web_ui/lib/painting.dart

```diff
@@ -332,10 +332,17 @@ abstract class Gradient extends Shader {
 }
 
 abstract class Image {
+  Image() {
+    onCreate?.call(this);
```

Comment:
The `Image` class is used as an interface, not as a super-class, so its constructor is not called. This should go in the concrete constructors of the `HtmlImage` and `CkImage` classes.

Also, the web engine has its own separate unit-tests, which is why this bug wasn't caught by tests. The tests are under `lib/web_ui/test`.